compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -695,6 +695,9 @@ pub trait PrettyPrinter<'tcx>: Printer<'tcx> + fmt::Write {
                 }
             }
             ty::FnPtr(ref sig_tys, hdr) => p!(print(sig_tys.with(hdr))),
+            ty::UnsafeBinder(ref bound_ty) => {
+                self.wrap_binder(bound_ty, |ty, cx| cx.pretty_print_type(*ty))?;
```

Comment:
Won't this produce a `for` binder?

Reply:
Yep. I thought I had a FIXME, but I geuss I forgot :)